docs/operator.md

```diff
@@ -1,168 +0,0 @@
-# Operator
-
-The RancherOS operator is responsible for managing the RancherOS versions
-and maintaining a machine inventory to assist with edge or baremetal installations.
-
-## Installation
-
-The RancherOS operator can be added to a cluster running Rancher Multi Cluster
-Management server.  It is a helm chart and can be installed as follows:
-
-```bash
-helm -n cattle-rancheros-operator-system install --create-namespace rancheros-operator https://github.com/rancher/os2/releases/download/v0.1.0-alpha12/rancheros-operator-0.1.0-alpha12-amd64.tgz
```

Comment:
That's our only front facing docs available. I would replace it for now with instructions on how to install the chart from the new location, or point to the documentation of the chart if there is any

Reply:
I will link to the ros-operator docs which are on their own repo :)